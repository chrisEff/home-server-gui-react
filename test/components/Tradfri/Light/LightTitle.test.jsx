const React = require('react')
const LightTitle = require('../../../../src/components/Tradfri/Light/LightTitle')

const {shallow} = require('enzyme')

describe('LightTitle', () => {
	it('should render correctly', () => {
		const wrapper = shallow(<LightTitle title='test' />)
		expect(wrapper).toMatchSnapshot()
	})
})
