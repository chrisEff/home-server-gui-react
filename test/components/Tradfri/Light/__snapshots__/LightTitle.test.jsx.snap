// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LightTitle should render correctly 1`] = `
<span
  className="name"
  onClick={[Function]}
>
  test
</span>
`;
